components/Feed/index.js

```diff
@@ -116,20 +178,37 @@ class Feed extends Component {
                 </Interaction.H1>
               )}
               {nodes &&
-                nodes.map(doc => (
-                  <TeaserFeed
-                    {...doc.meta}
-                    kind={
-                      doc.meta.template === 'editorialNewsletter' ? (
-                        'meta'
-                      ) : (
-                        doc.meta.kind
+                groupByDate.entries(nodes).map(({key, values}) =>
+                  <section>
+                    <div {...styles.header}>
+                      <StickyHeader>
+                        <span {...styles.date}>
```

Comment:
`styles.date` doesn't exist, so you can remove the whole `<span>` I guess.

Reply:
👌